packages/ui/src/primitives/Popover.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import * as Popover from '@radix-ui/react-popover'
+import { styled } from '../../stitches.config'
+import Box from './Box'
+
+const Arrow = styled(Popover.Arrow, {
+  width: 15,
+  height: 7,
+  fill: '$slate5',
+})
+
+const Content = styled(Popover.Content, {
+  filter: 'drop-shadow(0px 2px 16px rgba(0, 0, 0, 0.9))',
+})
+
+const RKPopover = ({ children, content }: any) => {
```

Comment:
Do we want to prefix our primitives with RK? If so we should do it across the board. I think it's a bit easier on the eyes not to prefix and the chances of clashing is lower.